spec/system/dashboard_system_spec.rb

```diff
@@ -5,30 +5,56 @@
 describe 'Dashboard', type: :system, js: true do
   include SystemHelpers
 
+  let(:credentials) { encryptables(:credentials1) }
+  let(:team) { teams(:team1) }
+
+
   it 'renders dashboard grid' do
     login_as_user(:alice)
+    visit("/encryptables/#{credentials.id}")
+    visit('/dashboard')
 
     expect(page.current_path).to eq('/dashboard')
-
     expect(page).to have_selector('pzsh-hero', visible: true)
     expect(page).to have_selector('pzsh-banner', visible: true)
+    expect(page).to have_text('Recent Credentials', count: 1)
     expect(page).to have_text('Favourites', count: 1)
     expect(page).to have_text('Teams', count: 1)
-
     expect(page).not_to have_selector 'div.content'
-
-    expect(page).to have_selector('div.dashboard-grid-card', count: 2)
+    expect(page).to have_selector('div.primary-content', count: 3)
+    expect(page).to have_selector('div.dashboard-grid-card', count: 3)
   end
 
   it 'navigates to team on team card click' do
     login_as_user(:alice)
+    visit("/encryptables/#{credentials.id}")
+    visit('/dashboard')
+
 
     expect(page.current_path).to eq('/dashboard')
 
     expect(page).to have_selector('pzsh-hero', visible: true)
-    all('div.dashboard-grid-card').first.click
 
-    expect(page.current_path).to eq('/teams/235930340')
+    first('div.dashboard-grid-card', text: team.name).click
+
+    expect(page.current_path).to eq("/teams/#{team.id}")
     expect(page).to have_selector 'div.content'
   end
+
+  it 'navigates to encryptable on encryptable card click' do
```

Comment:
```suggestion
  it 'lists recently accessed encryptable' do
```
very nice :partying_face: